samples/excel/90-just-for-fun/patterns.yaml

```diff
@@ -0,0 +1,210 @@
+id: excel-90-just-for-fun-patterns-
+name: Colorful Patterns
+description: Shows how to use range formatting to draw interesting pattern. Contributed by Alexander Zlatkovski
+author: AlexanderZlatkovski
+host: EXCEL
+api_set:
+    ExcelApi: 1.2
+script:
+    content: |-
+        $("#spiral").click(drawSpiral);
+        $("#squares").click(drawSquares);
+        $("#cross").click(drawCross);
+
+        async function drawSquares() {
+            try {
+                await Excel.run(async (context) => {
+                    const sheet = await OfficeHelpers.ExcelUtilities.forceCreateSheet(context.workbook, "Patterns", true /*clearOnly*/);
+                    sheet.activate();
```

Comment:
In fact, at that point, I think we can just say that the fix to Office Helpers is orthogonal to these samples (and we already have the exact same problem for all of our other samples anyway).

@jakobpn , could you go ahead and merge this in?  And I'll do the separate PR to update all samples to 0.7.2.  Filing bug https://github.com/OfficeDev/office-js-snippets/issues/31 to track the latter.